clang/test/Driver/sycl-offload-header-check.cpp

```diff
@@ -9,7 +9,8 @@
 // RUN: env TMP=%t_dirname  \
 // RUN: %clang -### -fsycl %s 2>&1 | \
 // RUN: FileCheck --check-prefix=CHECK-HEADER %s
+// CHECK-HEADER-NOT: non-portable path
```

Comment:
Just curious, how could one make this parameter sequence show up in the command?

Reply:
hah!  That's a good point.  The `non-portable path` output is from the FE that would be encountered after processing the generated integration header.  This is obviously not taking place here and the check is essentially not valid - I'll remove.